lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -101,6 +100,8 @@ def trigger
       puppet_trigger['start_date'] = self.class.normalized_date("#{trigger['start_year']}-#{trigger['start_month']}-#{trigger['start_day']}")
       puppet_trigger['start_time'] = self.class.normalized_time("#{trigger['start_hour']}:#{trigger['start_minute']}")
       puppet_trigger['enabled']    = trigger['flags'] & Win32::TaskScheduler::TASK_TRIGGER_FLAG_DISABLED == 0
+      puppet_trigger['minutes_interval'] = trigger['minutes_interval'] ||= 0
+      puppet_trigger['minutes_duration'] = trigger['minutes_duration'] ||= 0
```

Comment:
I think you may be confusing concepts. There are triggers that happen as the task schedule, then a repeat interval and duration after the task schedule trigger occurs. Perhaps it is better shown as a visualization? 

![image](https://cloud.githubusercontent.com/assets/63502/6966608/f2cc861c-d91f-11e4-911e-6e59e83f702f.png)